packages/terra-base/src/Base.jsx

```diff
@@ -70,7 +78,7 @@ class Base extends React.Component {
     const messages = Object.assign({}, this.state.messages, customMessages);
 
     if (locale === undefined) return childComponent;
-    if (!this.state.areTranslationsLoaded) return null;
+    if (!this.state.areTranslationsLoaded) return <div {...customProps}>{this.props.translationsLoadingPlaceholder}</div>;
```

Comment:
For this code, what's the reason of adding `customProps` to the placeholder's div? I think all style related props(class...) in customProps are specified for the loaded div.

I suggest we make the assumption, the loadingPlaceholder should be encapsulated with its own style config.